README.md

```diff
@@ -545,10 +545,10 @@ if (auto str = create(true)) {
 A type-safe container for single values of any type.
 ```c++
 std::any x {5};
-x.has_value() // == true
-std::any_cast<int>(x) // == 5
-std::any_cast<int&>(x) = 10;
-std::any_cast<int>(x) // == 10
+x.has_value(); // == true
+std::any_cast<int>(x); // == 5
+std::any_cast<int&>(x); = 10;
```

Comment:
Syntax error here.

Reply:
I have removed the syntax error in line 550

Reply:
That line should be `std::any_cast<int&>(x) = 10;`. The left side casts `x` to an `int&` and assigns it the value `10`. It's like if you wrote it out like this:
```cpp
int& x_ref = std::any_cast<int&>(x);
x_ref = 10;
```
Also, I forgot to mention to please make the change to CPP17.md as well. Sorry about that!

Reply:
The error is fixed